dozer/cogs/starboard.py

```diff
@@ -0,0 +1,366 @@
+"""Cog to post specific 'Hall of Fame' messages in a specific channel"""
+import logging
+import typing
+
+import discord
+from discord.ext.commands import guild_only, has_permissions
+
+from ._utils import *
+from .. import db
+
+MAX_EMBED = 1024
+DOZER_LOGGER = logging.getLogger('dozer')
+VIDEO_FORMATS = ['.mp4', '.mov', 'webm']
+
+
+async def is_cancelled(config, message, me, author=None):
+    """Determine if the message has cancellation reacts"""
+    if author is None:
+        author = message.author
+
+    for reaction in message.reactions:
+        if str(reaction) != config.cancel_emoji:
+            continue
+
+        users = await reaction.users().flatten()
+        if author in users or me in users:
+            await message.add_reaction(config.cancel_emoji)
+            return True
+
+    return False
+
+
+def make_starboard_embed(msg: discord.Message, reaction_count):
+    """Makes a starboard embed."""
+    e = discord.Embed(color=msg.author.color, title="New Starred Message")
```

Comment:
i mean this in the nicest way possible but what is with you people and unnecessarily wasting valuable embed title space with "New XYZ" for every single embed that will be sent in a channel with other embeds that say the exact same thing lol

Reply:
I mostly modeled these embeds off of the LGBTQ+ of FIRST server's (i.e. arynbot's) embeds. I think they look fine? I prefer this method to your original form for sure.